packages/data-point/examples/entity-model-before.js

```diff
@@ -0,0 +1,17 @@
+const dataPoint = require('../').create()
+const assert = require('assert')
+
+const toArray = acc => {
+  return Array.isArray(acc.value) ? acc.value : [acc.value]
+}
+
+dataPoint.addEntities({
+  'model:foo': {
+    before: toArray,
+    value: '$'
```

Comment:
It'd be interesting to use `inputType` and `outputType` to show that they can be different and expect a number and return an array.